packages/react-components/src/components/kpi/updatedKpiBase.tsx

```diff
@@ -31,6 +32,8 @@ export const UpdatedKpiBase: React.FC<KPIProperties> = ({
   };
 
   const point = propertyPoint;
+  const fontColor =
+    backgroundColor === '#ffffff' ? '' : highContrastColor(backgroundColor);
```

Comment:
use the default colors in design if background is white, otherwise use a high contrast font color for accessibility